src/interop/js-interop/package-web.md

```diff
@@ -0,0 +1,195 @@
+
+The `dart fix` should cover the majority of renames.
+If you come across a `dart:html` symbol without a built-in fix,
+first let us know by filing an [issue][].
+Then, you can manually discover the `package:web` name
+by looking up its `@Native` annotation.
+You can do this by either:
+
+- Ctrl or cmd clicking the name in the IDE and choosing **Go to Definition**.
+- Searching for the name in the [`dart:html` API docs][html] 
+  and checking its page under *Annotations*.
+
+### Type checks
+
+It's common for `dart:html` code to utilize runtime `is` checks
+to confirm whether an object actually maps to its JS type.
+In constrast, the types that represent browser bindings in `package:web`
+are not reified. This means a runtime `is` check can't confirm the interop type,
+only that the underlying type is JS `Object`.
+
+To be able to perform type checks, migrate any `dart:html` code
+using `is` type tests to use [interop methods][] like `instanceOf`.
+The [Compatibility, type checks, and casts][]
+section of the JS types page covers static interop alternatives to `is`
+in detail.
+
+```dart
+obj is Window; // Remove
+obj.instanceOf('Window'); // Add
+```
+
+### Method conversions
+
+To migrate unsupported method calls from `dart:html` to `package:web`,
+you'll mostly want to use conversion methods like `toJS()` and `toDart()`.
+Learn how to use static interop conversion methods from the [Conversions][]
+section of the JS types page.
+
+```dart
+_clickSubscription = window.onClick.listen(callback); // Remove
+window.onclick = callback.toJS; // Add
+```
+
+Generally, you can spot which methods need to migrate because they'll be flagged 
+with some variation of the exception:
+
+```nocode
+A value of type '...' can't be assigned to a variable of type 'JSFunction?'
+```
+
+Also important to note that methods are [statically dispatched][static interop]
+in `package:web`. So, unlike `dart:html`, there is no support for
+dynamic dispatch like virtual methods, overriding members, or mocks.
+
+[Helpers](#helper-layer) are available for some method call conversions
+(and are continuosly being added to). You can find available helpers in the
+[repo][helpers].
+
+### Zones
+
+In `dart:html`, calls are automatically zoned.
+This is not the case in `package:web`. There is no
+automatic binding of async callbacks in the current zone.
+
+To migrate, you can still to use zones,
+but you will have to [write them yourself][zones].
+There is no [helper](#helper-layer) available yet.
+
+## Helper layer
+
+The core of `package:web` provides direct interop with external
+members from the IDL, but does not provide solutions for some other
+functionality that `dart:html` handled by default.
+To mitigate these differences,
+`package:web` provides a [`helpers`][helpers] library for additional support
+in handling a number of use cases that aren't directly
+available through the core interop.
+The helper library contains classes, typedefs, and extension methods
```

Comment:
I think we're actually planning on removing this now that we have the dart fix. :)

I'd maybe reword this sentence as "The helper library contains various types and members to expose some legacy features that exist within the Dart web libraries."